baseline.py

```diff
@@ -49,27 +48,32 @@ def generate_clustering(self, num_to_review):
     def evaluate_baseline(self, sim, out_prefix, n_min, n_max, n_inc=10):
         gt_results = []
         r_results = []
-        for n in range(n_min, n_max+1, n_inc):
+        for n in range(n_min, n_max + 1, n_inc):
             clustering, node2cid = self.generate_clustering(n)
-            result = sim.incremental_stats(n, clustering, node2cid,
-                                           sim.gt_clustering, sim.gt_node2cid)
+            result = sim.incremental_stats(
+                n, clustering, node2cid, sim.gt_clustering, sim.gt_node2cid
+            )
             gt_results.append(result)
-            result = sim.incremental_stats(n, clustering, node2cid,
-                                           sim.r_clustering, sim.r_node2cid)
+            result = sim.incremental_stats(
+                n, clustering, node2cid, sim.r_clustering, sim.r_node2cid
+            )
             r_results.append(result)
 
-        out_name = out_prefix + "_gt.pdf"
+        # TODO: Added by JP due to linting errors, is this correct?
+        sim_i = sim
```

Comment:
This code block refers to a `sim_i` variable that is not defined anywhere in the context above.  Should it be renamed as simply `sim` on lines 70 and 71?